### ISSUE TITLE
jq output wrong numbers if it meets big numbers.

### ISSUE THREAD

**Issue by Clcanny** [stedolan/jq#2319]
**Describe the bug**

jq outputs wrong numbers if it meets wrong numbers.

**To Reproduce**

```bash
# cat test.json
{
  "id1": 18014398509481995,
  "id2": 18014398509481994
}
# cat test.json | jq
{
  "id1": 18014398509481996,
  "id2": 18014398509481990
}
```

**Expected behavior**

js outputs the same numbers.

```bash
# cat test.json | jq
{
  "id1": 18014398509481996,
  "id2": 18014398509481990
}
```

**Environment (please complete the following information):**

```bash
# cat /etc/os-release | head -n 1
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
# jq --version
jq-1.5-1-a5b5cbe
```

**Additional context**

**Comment by Clcanny** [comment]
It seems this problem is fixed in master.

**Comment by victor-yarema** [comment]
Seems like needed feature was implemented in cf4b48c7ba30cb30e116b523cff036ea481459f6 .

**Comment by Clcanny** [comment]
Thanks, I will close this issue. I think publishing a version that fix this bug is a good idea.
